frontend/tests/playwright/product/disabled_product.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { test, expect } from '@playwright/test';
+
+test.describe('Disabled Product Test', () => {
+   test('Not for Sale text renders and noindexed', async ({ page }) => {
+      await page.goto('products/iphone-6s-plus-replacement-battery-disabled');
+
+      await expect(page.getByText('Not for Sale')).toBeVisible();
+      await expect(page.getByText('Description')).toBeVisible();
```

Comment:
In the original issue, https://github.com/iFixit/react-commerce/issues/1153, it says `no variant details` doesn't this mean that this should `not.toBeVisible()`?

Reply:
Ah, scratch that. I see that there is some elaboration later in the issue `We will keep the images, description, warranty, video, etc.`

Reply:
We should confirm that `assembly contents` and `kit contents` are not visible then.